config/src/main/java/org/mobilitydata/gtfsvalidator/config/DefaultConfig.java

```diff
@@ -143,4 +141,12 @@ public LogExecutionInfo logExecutionInfo() {
     public PrintHelp printHelp() {
         return new PrintHelp(execParamRepo, new HelpFormatter());
     }
+
+    public ProcessParsedAgency processParsedAgency() {
+        return new ProcessParsedAgency(resultRepo, gtfsDataRepository, new Agency.AgencyBuilder(new ArrayList<>()));
```

Comment:
After discussion, we decided to remove parametric constructors from our builder classes

Reply:
See [related PR](https://github.com/MobilityData/gtfs-validator/pull/187)